org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/handler/ZigBeeCoordinatorHandler.java

```diff
@@ -439,6 +445,13 @@ private synchronized void initialiseZigBee() {
                 return;
         }
 
+        // Add all the clusters that we are supporting.
+        // If we don't do this, the framework will reject any packets for clusters we have not stated support for.
+        channelFactory.getImplementedClientClusters().stream()
```

Comment:
According to the OSGi references in the handler factories the channelFactory can be `null`.
This would throw a NPE here.